README.md

```diff
@@ -4,9 +4,9 @@ testing GH merge PR bugs
 
 Alpha
 
-Beta
+Beta updated
```

Comment:
This line is identical to the one in master, but I have extra stuff to review just because of the wrong master being compared to pull